src/components/Input.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Input = ({...props}) => (
```

Comment:
no need for using spread operator here, you can do this:

const Input = (props) =>

Reply:
It doesn't work when I change it to just props.

Reply:
Then you're not doing something right, probably mutating props somewhere